src/globus_cli/commands/gcs/endpoint/role/list.py

```diff
@@ -0,0 +1,42 @@
+import uuid
+
+import click
+from globus_sdk.paging import Paginator
+
+from globus_cli.commands.gcs.endpoint.role._common import role_fields
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import command, endpoint_id_arg
+from globus_cli.services.transfer import iterable_response_to_dict
+from globus_cli.termio import display
+from globus_cli.utils import PagingWrapper
+
+
+@command(
+    "show",
```

Comment:
```suggestion
    "list",
```
typop

Reply:
Can you guess which command I wrote first 😬